### PR TITLE
Enable coin.space.xml

### PR DESCRIPTION
**Pull request by mahnunchik**
Enable by default coin.space.xml

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring existing code
- [x] New Ruleset
- [x] Existing Ruleset

### CODE REVIEW

**Review by J0WI — COMMENTED** [review]
We [prefer listing explicit target hosts](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#left-wildcards). Is there a strong reason to use a wildcard here?

**Comment by mahnunchik** [comment]
Hi @J0WI Yep, there is strong reason to use wildcard domains for crypto nodes like `xrp.coin.space` and `xlm.coin.space`. That domains can be added and removed at any time for example in case of emergency fork of cryptocurrency.

**Comment by pipboy96** [comment]
@mahnunchik Hello and thanks for contributing! For the record, are you an owner or a member of a coin.space team? If yes, you may still submit the PR, but it's recommended that you use [HSTS preloading](https://hstspreload.org/) instead.

File: src/chrome/content/rules/coin.space.xml [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
 	<target host="coin.space" />
-	<target host="www.coin.space" />
-	<target host="db.coin.space" />
-	<target host="live.coin.space" />
-	<target host="proxy.coin.space" />
-	<target host="ru.coin.space" />
+	<target host="*.coin.space" />
+	<test url="http://www.coin.space/" />
+	<test url="http://xrp.coin.space/api/v1/" />
+	<test url="http://xlm.coin.space/api/v1/" />
```
Comment (J0WI) [inline src/chrome/content/rules/coin.space.xml:6]: Cloud you add a comment with a hint that there is a subdomain for every cryptocurrency, so we are aware of this behavior and the requirement of the wildcard rule.

Comment (nikashitsa) [reply src/chrome/content/rules/coin.space.xml:6]: Hi! @mahnunchik is member of coin.space team. You can check it here https://github.com/CoinSpace/CoinSpace/graphs/contributors

**Comment by mahnunchik** [comment]
@J0WI comment have been added.

@pipboy96 we are targeting Tor browser in most. So it would be helpful to have that rule.

**Review by J0WI — APPROVED** [review]

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**